### ISSUE TITLE
Microsoft.AspNetCore.AngularServices with MVC5

### ISSUE THREAD

**Issue by naveedahmed1** [aspnet/JavaScriptServices#106]
Is it possible to use Microsoft.AspNetCore.AngularServices with ASP.Net MVC5 application?

**Comment by laskoviymishka** [comment]
Not yet, it require NET451 build (which is still pending PR)

**Comment by naveedahmed1** [comment]
Thank you for the update Andrei, should we expect support for MVC5 in near future? Or should it take time.

**Comment by laskoviymishka** [comment]
**TL/DR**
It's possible but really hard

@naveedahmed1 well it quite big task, so after net451 PR be merged it need to make some research araound possibilities of such integration. Some features will be unavaible (like tag-helpers) becouse we don't have such stuff in MVC5. Basically it may require some amount of time to extract something that could be used in MVC5. Right now every package of this require some of aspnet core things. Most lightweight is `NodeServices` which require only 
```
 "Microsoft.AspNetCore.Hosting.Abstractions": "1.0.0-*",
    "Microsoft.Extensions.Configuration.Json": "1.0.0-*",
    "Microsoft.Extensions.DependencyInjection.Abstractions": "1.0.0-*",
    "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
```
If this could be added to your mvc5 project then `Nodeservices` will be fine. For `SpaServices` and `AngularServices` and `ReactServices` it could be more difficult since them depend on ` "Microsoft.AspNetCore.Mvc": "1.0.0-*"`
So it possible but really hard :sweat: Any help would be great with these one :+1:

**Comment by SteveSandersonMS** [comment]
As @laskoviymishka mentions, the core `NodeServices` library should work perfectly well when referenced from any .NET 4.5.1 app (including an MVC 5 app), but there's more to do if you want to have the same or similar easy APIs for interacting with it like you do on .NET Core.

If someone wants to contribute some kind of wrapper library for consuming `NodeServices` features on ASP.NET MVC 5, that would be very interesting. Please let me know if so - I'd be open to integrating it here.

I'll close this now as it's not something that I'll be working on because of other priorities. Again, if someone does demonstrate a good solution here, we can definitely discuss it!
